testing/unit/test_configurator.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for configurator"""
+
+import unittest
+import os
+
+from daq.configurator import Configurator, print_config
+
+def dict_to_str(obj):
+    """Dump dict as key=value to str object."""
+    config_list = []
+    for key in sorted(obj.keys()):
+        value = obj[key]
+        quote = '"' if ' ' in str(value) else ''
+        config_list.append("%s=%s" % (key, obj[key]))
+    return '\n'.join(config_list)
+
+
+class TestConfigurator(unittest.TestCase):
+    """Test class for Configurator"""
+
+    config = {
+        'monitor_scan_sec' : '30',
+        'default_timeout_sec' : '350',
+        'base_conf' : 'misc/module_config.json',
+        'site_path' : 'local/site/',
+        'initial_dhcp_lease_time' : '120s',
+        'dhcp_lease_time' : '500s',
+        'long_dhcp_response_sec' : '105'
+    }
+
+    def setUp(self):
+        tmpfile = open('temp.conf', 'w+')
```

Comment:
extract "temp.conf" to a constant variable since it's used in 3x places

Reply:
Addressed.